README.md

```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
 
 The following subprojects are owned by the Working Group:
 
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
```

Comment:
Is this being kept up to date at all? Last update was 2015 - maybe it should be deprecated (or, maybe it needs to be updated)

Reply:
potentially good to maintain with a robot webtool usage tutorial update. Otherwise, let's deprecate it